datasets/sentinel-5p/collection/template.json

```diff
@@ -0,0 +1,238 @@
+            "href": "https://sentinel.esa.int/documents/247904/690755/Sentinel_Data_Legal_Notice",
+            "type": "application/pdf",
+            "title": "Sentinel Data License"
+        },
+        {
+            "rel": "about",
+            "href": "https://sentinel.esa.int/web/sentinel/missions/sentinel-5p",
+            "type": "text/html",
+            "title": "Sentinel-5 Precursor Mission"
+        }
+    ],
+    "stac_extensions": [
+        "https://stac-extensions.github.io/sat/v1.0.0/schema.json",
+        "https://stac-extensions.github.io/table/v1.2.0/schema.json",
+        "https://stac-extensions.github.io/item-assets/v1.0.0/schema.json"
+    ],
+    "keywords": [
+        "ESA",
+        "Copernicus",
+        "Sentinel",
+        "Air Quality",
+        "Climate Change",
+        "Forecasting"
+    ],
+    "msft:short_description": "Sentinel-5P Level 2 atmospheric monitoring products in NetCDF format",
+    "msft:storage_account": "sentinel5euwest",
+    "msft:container": "sentinel-5p",
+    "msft:region": "westeurope",
+    "providers": [
+        {
+            "name": "ESA",
+            "roles": [
+                "producer",
+                "processor",
+                "licensor"
+            ],
+            "url": "https://earth.esa.int/web/guest/home"
+        },
+        {
+            "name": "Microsoft",
+            "roles": [
+                "host"
+            ],
+            "url": "https://planetarycomputer.microsoft.com"
+        }
+    ],
+    "assets": {
+        "thumbnail": {
+            "title": "Sentinel-5P Level-2 NetCDF Thumbnail",
+            "href": "https://ai4edatasetspublicassets.blob.core.windows.net/assets/pc_thumbnails/sentinel-5p-l2-netcdf-thumb.png",
+            "media_type": "image/png"
+        },
+        "geoparquet-items": {
+            "href": "abfs://items/sentinel-5p-l2-netcdf.parquet",
+            "title": "GeoParquet STAC Items",
+            "description": "Snapshot of the collection's STAC items exported to GeoParquet format.",
+            "type": "application/x-parquet",
+            "roles": [
+                "stac-items"
+            ],
+            "table:storage_options": {
+                "account_name": "pcstacitems"
+            },
+            "msft:partition_info": {
+                "is_partitioned": true,
+                "partition_frequency": "MS"
+            }
+        }
+    },
+    "summaries": {
+        "constellation": [
+            "Sentinel-5P"
+        ],
+        "platform": [
+            "Sentinel 5 Precursor"
+        ],
+        "instruments": [
+            "TROPOMI"
+        ],
+        "sat:platform_international_designator": [
+            "2017-064A"
+        ],
+        "s5p:collection_identifier": [
+            "01",
+            "02",
+            "03"
+        ],
+        "s5p:processing_mode": [
+            "NRTI",
+            "OFFL",
+            "RPRO"
+        ],
+        "s5p:product_type": [
```

Comment:
Just to confirm, is this the attribute we expect users to use for filtering products?

It *might* be worth adding a "friendly" product type (maybe call it `s5p:product_name`?) that strips things down a bit (but still keep this one too). I would definitely want to remove the underscroes. Maybe remove `L2`. So

- `AER_AI`
- AER_LH
- CH4
- CLOUD
- CO
- ...

Reply:
Yes, that was the expectation. But I see your point about that being awkward. I don't have a problem with the stripped down designation. I'll add it.

We should probably do the same for sentinel-3 where we have properties like `"s3:product_type": "SY_2_AOD___"`. In this case we would add an `s3:product_name": "AOD"` property.

Reply:
Done.